addons/fleet/models/fleet_vehicle.py

```diff
@@ -77,7 +77,7 @@ def _get_default_state(self):
     @api.depends('model_id', 'license_plate')
     def _compute_vehicle_name(self):
         for record in self:
-            record.name = record.model_id.brand_id.name + '/' + record.model_id.name + '/' + (record.license_plate or 'No Plate')
+            record.name = record.model_id.brand_id.name + '/' + record.model_id.name + '/' + (record.license_plate or _('No Plate'))
```

Comment:
@tivisse I believe `No license plate` is clearer than `No plate`? Especially if you have this open in Transifex and have no context.